client/src/site-search/search-results.tsx

```diff
@@ -169,6 +175,30 @@ export default function SearchResults({
   );
 }
 
+function SortOptions() {
+  const [searchParams] = useSearchParams();
+  const querySort = searchParams.get("sort") || SORT_OPTIONS[0][0];
+  return (
+    <p className="sort-options">
+      <b>Sort by</b>{" "}
```

Comment:
`<b>` is a presentational element. Shall we still use them?
(I recall HTML5 changed the semantics away from `bold` to … something else)

Reply:
If they are not deprecated or invalidate the document it would be the case to be able to use any element